phases/register.go

```diff
@@ -125,6 +129,20 @@ func Register(token string, progress Progress) error {
 		}
 	}
 
+	{
```

Comment:
if params.CodesignIOS.certificates || params.CodesignIOS.provisioningProfiles NOT empty